src/main/java/com/coveros/selenified/services/Response.java

```diff
@@ -163,13 +165,15 @@ public void assertEquals(JsonArray expectedArray) {
      * @param expectedMessage - the expected response message
      */
     public void assertEquals(String expectedMessage) {
-        Success success = Success.FAIL;
+        boolean pass = false;
         if (message != null) {
-            success = message.equals(expectedMessage) ? Success.PASS : Success.FAIL;
+            pass = message.equals(expectedMessage) ? true : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "true" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1125)